spec/models/experimentation/current_patient_experiment_spec.rb

```diff
@@ -30,6 +30,26 @@
   end
 
   describe "#memberships_to_notify" do
+    it "return memberships who were enrolled at any time of the day in the local timezone" do
+      experiment = create(:experiment, experiment_type: "current_patients")
+      treatment_group = create(:treatment_group, experiment: experiment)
+      create(:reminder_template, treatment_group: treatment_group, remind_on_in_days: -2)
+      patient_1 = create(:patient)
+      patient_2 = create(:patient)
+      Time.use_zone(CountryConfig.current[:time_zone]) do
+        treatment_group.enroll(patient_1, expected_return_date: 2.days.from_now.beginning_of_day)
+      end
+      Time.use_zone(CountryConfig.current[:time_zone]) do
+        treatment_group.enroll(patient_2, expected_return_date: 2.days.from_now.middle_of_day)
+      end
+      membership_1 = experiment.treatment_group_memberships.find_by(patient_id: patient_1.id)
+      membership_2 = experiment.treatment_group_memberships.find_by(patient_id: patient_2.id)
+
+      expect(membership_1.expected_return_date.to_date).to eq(1.day.from_now.to_date)
+      expect(membership_2.expected_return_date.to_date).to eq(2.day.from_now.to_date)
```

Comment:
thinking if i should remove these:
- they'll fail in case the configured tz for tests is not IST
- but i added them to make it easy to understand why the fix is necessary
- the configured tz might not change (check this)
- this is a temporary fix, though unsure when we'll pick up the longterm fix of changing the field types from timestamp to date

Reply:
Set this to Asia/Kolkata so it's not dependednt on the country config.